frontend/src/pages/auth/components/sign-up-form/styles.module.scss

```diff
@@ -0,0 +1,43 @@
+.auth-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 280px;
+}
+
+.auth-form .greeting {
+  margin: 0;
+  font-weight: var(--semibold);
+  font-size: 30px;
+}
+
+.auth-form .btn {
```

Comment:
these styles should be in button component itself except margin

Reply:
done